geopandas/tests/test_crs.py

```diff
@@ -393,6 +393,44 @@ def test_dataframe_setitem_without_geometry_column(self):
         df["geometry"] = self.geoms
         assert df.crs is None
 
+    def test_dataframe_multiindex_cols_2level(self):
```

Comment:
This probably shouldn't live in test_crs. It is primarily an issue with the specification of the active geometry. It affects the .crs attribute only as a result of that.

Reply:
I've moved the test to test_geodataframe::TestConstructor - next to another multiindex test.